src/Sulu/Bundle/SecurityBundle/Admin/SecurityAdmin.php

```diff
@@ -110,6 +110,7 @@ public function getRoutes(): array
                 ->addOption('tabTitle', 'sulu_security.permissions')
                 ->addOption('backRoute', 'sulu_contact.contacts_datagrid')
                 ->addOption('resourceKey', 'users')
+                ->addOption('idQueryParameter', 'contactId')
```

Comment:
I also don't know if it will be too many parameters here, if we also add another parameter for getting the parameter from the other direction (e.g. when loading the excerpt from an entity).